manifest/com.Dexy/CustomPlatformColors/info.json

```diff
@@ -0,0 +1,28 @@
+{
+	"name": "CustomPlatformColors",
+	"id": "com.Dexy.CustomPlatformColors",
+	"description": "Change your UI and UIX colors, configurable through mod settings. 🎨",
+	"category": "Visual Tweaks",
+	"sourceLocation": "https://github.com/DexyThePuppy/CustomPlatformColors",
+	"additionalAuthors": [
+		"troyBORG",
+		"SectOLT",
+		"LeCloutPanda",
+		"SectOLT"
+	],
+	"tags": [
+		"colour",
+		"interface",
+		"color"
+	],
+	"versions": {
+		"1.2.0": {
+			"releaseUrl": "https://github.com/DexyThePuppy/CustomPlatformColors/releases/tag/v1.2",
+			"artifacts": [{
+				"url": "https://github.com/DexyThePuppy/CustomPlatformColors/releases/download/v1.2/CustomPlatformColors.dll",
```

Comment:
Should use the modloader provided [logging methods](https://github.com/resonite-modding-group/ResoniteModLoader/wiki/Creating-Mods#logging) instead of `UniLog.Error()` directly. Doing so will also handling adding the mod name and the level logged so you don't need to.